source/sdk/Time.ooc

```diff
@@ -9,8 +9,7 @@
 import os/win32
 
 version(linux) {
-	include unistd | (__USE_BSD), sys/time | (__USE_BSD), time | (__USE_BSD)
-	//include unistd, sys/time, time
+	include unistd | (__USE_BSD, _BSD_SOURCE), sys/time | (__USE_BSD, _BSD_SOURCE), time | (__USE_BSD, _BSD_SOURCE)
```

Comment:
According to http://man7.org/linux/man-pages/man7/feature_test_macros.7.html : 
```
_BSD_SOURCE (deprecated since glibc 2.20)
...
Since glibc 2.20, this macro is deprecated.  It now has the same effect as defining _DEFAULT_SOURCE, but generates a compile-time warning (unless _DEFAULT_SOURCE is also defined).
To allow code that requires _BSD_SOURCE in glibc 2.19 and earlier and _DEFAULT_SOURCE in
glibc 2.20 and later to compile without warnings, define both _BSD_SOURCE and _DEFAULT_SOURCE.
```
can you try with both macros ?